src/Gaffer/BackgroundTask.cpp

```diff
@@ -329,22 +329,3 @@ void BackgroundTask::cancelAffectedTasks( const GraphComponent *actionSubject )
 		it = nextIt;
 	}
 }
-
-void BackgroundTask::cancelAllTasks()
```

Comment:
Why don't we need this anymore? The name makes it sound generally useful, even if Monitor isn't using it anymore. Or was it fundamentally flawed?

Reply:
It existed only to support the old workaround that is sorted out properly by this commit. It was private, and wasn't used anywhere else. We can bring it back if we need it one day, but there's no point maintaining dead code.